vagrant.md

```diff
@@ -63,6 +63,13 @@ To destroy all VMs
 vagrant(.exe) destroy
 ```
 
+In case vagrant needs to be set using a private network due to issues in getting IPs in the public network the option **--network-type=private_network** should be used.
+```
+vagrant.exe --network-type=private_network up
+vagrant.exe --network-type=private_network reload
```

Comment:
It was an example showing how to run the command up and reload. 
Indeed it is confusing and I will fix it.